packages/runtime/src/store/index.ts

```diff
@@ -351,10 +373,7 @@ export class TutorialStore {
     }
   }
 
-  updateFiles(files: Files) {
-    this._runner.updateFiles(files);
-  }
-
```

Comment:
This and `refreshStyles` below are unused code. I though we might as well just remove them.